.github/workflows/python-integration-tests.yml

```diff
@@ -81,6 +81,13 @@ jobs:
         run: |
           ollama pull ${{ vars.OLLAMA_MODEL }}
           ollama list
+      - name: Google auth
+        uses: google-github-actions/auth@v2
+        with:
+          project_id: ${{ vars.VERTEX_AI_PROJECT_ID }}
+          credentials_json: ${{ secrets.VERTEX_AI_SERVICE_ACCOUNT_KEY }}
```

Comment:
Not blocking for this PR, but we should have a discussion about how we store secrets for tests like these. We may need to come up with a different plan.

Reply:
Do you mean how we internally manage the secrets?